doc-site/docs/components/kubernetes/user-auth.md

```diff
@@ -45,7 +45,46 @@ The default setup uses an Authenticator handler: [`cookie_session`][oathkeeper-c
 - Oathkeeper rules
 - jwks_private_key (Oathkeeper uses this key to sign session tokens)
 
-## Documentation
+## Configuring
+The default values can be overriden using the variable `kratos_values_override` and `oathkeeper_values_override`. You can pass in an object that is a subset of the Kratos or Oathkeeper config in the same nesting level and it will merge with the default values.
```

Comment:
This is actually the helm chart config right? How about a link to the helm chart values so people can see all the possible options?

Reply:
what helm chart doesnt really show the values, 
 https://github.com/ory/k8s/blob/master/helm/charts/kratos/values.yaml#L89 
i linked the https://github.com/commitdev/zero-aws-eks-stack/pull/217/files#diff-25dd24bdd30f2b5ade0d10f6d4d147456f340f1e3693f9804d16980a61397bdfR85 instead where its better documented what are the possible values, the entire thing just get mounted as a volume `/etc/config/kratos.yaml` i believe